Cargo.toml

```diff
@@ -23,7 +23,7 @@ cortex-m = "0.6"
 cortex-m-rt = "0.6"
 embedded-hal = "0.2"
 nb = "0.1"
-stm32f3 = "0.10"
+stm32f3 = "0.11.0"
```

Comment:
For consistency, I should use "0.11". Thanks for pointing this out :) 

Cargo's version always translates to the "^0.11.0" [semver](https://semver.org/) version number, unless stated otherwise. So it should not make a difference, as `0.11.1` would automatically be pulled in, when such a version would be released.